src/main/kotlin/dsm/wemeet/global/socket/domain/RoomWebSocketHandler.kt

```
@@ -79,15 +93,20 @@ class RoomWebSocketHandler(
             "kick" -> {
                 val currentEmail = getUserEmail(session)
 
+                println("kick 메시지 받았으")
+
                 try {
                     kickMemberUseCase.execute(roomId, currentEmail, signal.to!!)
                 } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)